features/org.wso2.carbon.appmgt.store.feature/src/main/resources/admin-dashboard/site/themes/default/templates/businessowners-new/js/businessowners-new.js

```diff
@@ -116,20 +117,27 @@ $(document).on("click", "#btn-owner-save", function () {
     var ownerMail = $('#businessOwnerEmail').val();
     var description = $('#businessOwnerDesc').val();
     var siteLink = $('#businessOwnerSite').val();
-    var ownerDetails = {};
+    var ownerProperties = {};
 
     if(fieldCount > 0){
         var i = fieldCount;
         while(i > 0){
             var key_id = "#key-".concat(i-1);
             var val_id = "#value-".concat(i-1);
+            var showInStoreId = "showInStore-".concat(i-1);
             var key = $(key_id).val();
-            var value = $(val_id).val();
-            ownerDetails[key] = value;
+            var value = [];
+            value.push($(val_id).val());
+            if(document.getElementById(showInStoreId).checked) {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5